src/utils/validations.js

```diff
@@ -23,23 +23,37 @@ export const validateEmail = (email) => {
 
 // Function to validate password with conditions
 export const validatePassword = (password) => {
-  const specialChars = /^(?=.*[!@#$%^&*])/;
+  const specialChars = /^(?=.*[!@#$_~\-%^&*])/;
   const numbers = /^(?=.*[0-9])/;
-  const forbiddenChars = /^(?=.*[(){}[\]|`¬¦"'<>:;|~_\-+=,])/;
+  const forbiddenChars = /^(?=.*[(){}[\]|`¬¦"'<>:;|+=,])/;
+  const passwordErrors = [];
 
   if (forbiddenChars.test(password)) {
-    return "Password contains invalid characters!";
-  } else if (password.length < 8) {
-    return "Password must be at least 8 characters long!";
-  } else if (
+    passwordErrors.push("Password contains invalid characters!", <br />);
+  }
+  if (password.length < 8) {
+    passwordErrors.push("Password must be at least 8 characters long!", <br />);
+  }
+  if (
     password.toLowerCase() === password ||
     password.toUpperCase() === password
   ) {
-    return "Password must contain at least 1 uppercase and 1 lowercase letter.";
-  } else if (!numbers.test(password)) {
-    return "Password must contain at least 1 number.";
-  } else if (!specialChars.test(password)) {
-    return "Password must contain at least 1 special character.";
+    passwordErrors.push(
+      "Password must contain at least 1 uppercase and 1 lowercase letter.",
+      <br />
+    );
+  }
+  if (!numbers.test(password)) {
+    passwordErrors.push("Password must contain at least 1 number.", <br />);
+  }
+  if (!specialChars.test(password)) {
+    passwordErrors.push(
+      "Password must contain at least one of the following: !@#$-_~%^&*",
+      <br />
```

Comment:
I would prefer to create line breaks between wherever `passwordErrors` gets rendered rather than here